src/TestSuite/IntegrationTestTrait.php

```diff
@@ -223,25 +223,9 @@ public function tearDown()
         $this->_retainFlashMessages = false;
     }
 
-    /**
-     * Toggle whether or not you want to use the HTTP Server stack.
-     *
-     * @param bool $enable Enable/disable the usage of the HTTP Stack.
-     * @return void
-     */
-    public function useHttpServer(bool $enable): void
-    {
-        if ($enable === false) {
-            deprecationWarning('Calling `useHttpServer(false)` does nothing, and will be removed.');
```

Comment:
We previously had talked about leaving this in to make upgrading easier. This is a pretty simple thing to remove with rector though if we want to go down that path.

Reply:
Not really a fan of keeping a no-op method. All people have to do is remove a statement.

Reply:
Ok.

Reply:
@markstory I add `IntegrationTestCase::useHttpServer()` instead and most upgrading from 3.x would be still extending that class instead of directly using this trait.